content/sensu-go/6.8/catalog/catalog-api.md

```diff
@@ -0,0 +1,474 @@
+---
+title: "Catalog API"
+description: "Use the Sensu Catalog API to generate static API content that the web UI can consume. Build your own private catalog of Sensu integrations."
+product: "Sensu Go"
+version: "6.8"
+weight: 60
+layout: "single"
+toc: true
+menu:
+  sensu-go-6.8:
+    parent: catalog
+---
+
+{{% notice commercial %}}
+**COMMERCIAL FEATURE**: Access the Catalog API in the packaged Sensu Go distribution.
+For more information, read [Get started with commercial features](../../commercial/).
+{{% /notice %}}
+
+{{% notice note %}}
+**NOTE**: The Sensu Catalog is in public preview and is subject to change.
+{{% /notice %}}
+
+The Catalog API generates a static API from a repository of integrations, such as https://github.com/sensu/catalog, via the [catalog-api command line tool][1].
+
+## Get the latest catalog SHA-256 checksum
+
+Retrieves the latest catalog content version's SHA-256 checksum.
+The Sensu web UI uses the checksum information to determine the latest API subpath.
+
+### Example
+
+The following example queries the Sensu Catalog API for the latest content version:
+
+{{< code shell >}}
+curl -X GET \
+/version.json
+{{< /code >}}
+
+The request returns the latest catalog content version's SHA-256 checksum and the time of the last update (in seconds since the Unix epoch):
+
+{{< code text >}}
+{
+  "release_sha256": "af3c54b86b90fac8977f1bdc80d955002dd3f441bdbb4cc603c94abbb929dcf6",
+  "last_updated": 1643664852
+}
+{{< /code >}}
+
+### API Specification
+
+/version.json (GET)  |     |
+---------------------|------
+description          | Retrieves the latest content version's SHA-256 checksum, which the Sensu web UI uses to determine the latest API subpath. Also returns the time of the last update in seconds since the Unix epoch.
+endpoint             | /version.json
+output               | {{< code text >}}
+{
+  "release_sha256": "af3c54b86b90fac8977f1bdc80d955002dd3f441bdbb4cc603c94abbb929dcf6",
+  "last_updated": 1643664852
+}
+{{< /code >}}
+response codes       | **PLACEHOLDER**
```

Comment:
Are there response and error codes for the catalog API?

Reply:
Kind of. The API is statically generated and thus the only error codes that can exist are errors from whichever webserver is serving the content (E.g. a 404 if the file doesn't exist or a 500 if there's some internal server error). This shouldn't happen unless there is a configuration issue with the webserver.